packages/vega-label/src/Label.js

```diff
@@ -86,7 +86,7 @@ inherits(Label, Transform, {
 
     // run label layout
     labelLayout(
-      pulse.materialize(pulse.SOURCE).source,
+      pulse.materialize(pulse.SOURCE).source || [],
       _.size,
       _.sort,
       array(_.offset || 1),
```

Comment:
```suggestion
      array(_.offset ?? 1),
```

Otherwise, an offset of 0 does not work, right?

Reply:
Yes, you are correct.